mage_ai/data_preparation/templates/transformers/transformer_actions/row/drop_duplicate.py

```diff
@@ -0,0 +1,24 @@
+from mage_ai.data_cleaner.transformer_actions.base import BaseAction
+from mage_ai.data_cleaner.transformer_actions.constants import ActionType, Axis
+from mage_ai.data_cleaner.transformer_actions.utils import build_transformer_action
+from pandas import DataFrame, concat
+
+if 'transformer' not in globals():
+    from mage_ai.data_preparation.decorators import transformer
+
+
+@transformer
+def execute_transformer_action(df: DataFrame, *args, **kwargs) -> DataFrame:
+    """
+    Execute Transformer Action: ActionType.DROP_DUPLICATE
+    """
+    concat((df, df))
```

Comment:
what's this?

Reply:
Leftover code from testing, I removed it